pkg/aws/apis/validation/validation.go

```diff
@@ -219,3 +242,11 @@ func contains(arr []string, checkValue string) bool {
 	}
 	return false
 }
+
+func validateStringValues(fld *field.Path, s string, accepted []string) field.ErrorList {
```

Comment:
This function could have just returned `Error` instead of `ErrorList` as there cannot be more than one error anyways.

Reply:
True, but I think the `ErrorList` is much more convenient to consume as a type. You can simply do an `append` to merge your error. Otherwise your signature needs to return `*field.Error` and need additional `nil` checks for the consumer of the `ErrorList`.